components/Chart.tsx

```diff
@@ -29,144 +41,110 @@ const Chart = (props: Props) => {
-      style: {
-        fontFamily: "inherit",
-        fontWeight: 700,
-      },
-    },
-    plotOptions: {
-      series: {
-        stacking: "normal",
-        borderWidth: 0,
-        pointPadding: 0,
-        groupPadding: 0,
-        animation: false,
-        enableMouseTracking: false,
-        colorByPoint: true,
-      },
-    },
-    series: series,
-    colors: categoryColors,
-    xAxis: [{
-      categories: categories,
-      lineColor: "transparent",
-      reversed: true,
-      labels: {
-        enabled: showLabels,
-        style: {
-          color: "black",
-          fontSize: "1em",
-        },
-      },
-      accessibility: {
-        description: "Obligations"
-      },
-    }, { // mirror axis on right side
-      opposite: true,
-      reversed: true,
-      linkedTo: 0,
-      visible: false,
-      accessibility: {
-        description: "Rights"
-      },
-    }],
-    yAxis: [{
-      title: {
-        text: ""
-      },
-      labels: {
-        enabled: false,
-      },
-      accessibility: {
-        description: "Intensity of Rights or Obligations",
-        rangeDescription: "0-5",
-      },
-      maxPadding: 0,
-      visible: true, // visible = true to show plotLines, but essentially hidden
-      min: -5,
-      max: 5,
-      tickInterval: 1,
-      gridLineColor: "transparent",
-      plotLines: [{
-        value: 0,
-        color: "#000",
-        width: 4,
-        zIndex: 999,
-      }],
-    }],
-    title: { text: "" },
-    tooltip: { enabled: false },
-    legend: { enabled: false },
-    credits: { enabled: false },
-    accessibility: {
-      point: {
-        valueDescriptionFormat: "{xDescription} ranks {value}/5 in"
-      }
-    },
-    responsive: {
-      rules: [{
-        condition: {
-          maxWidth: 400,
-        },
-        chartOptions: {
-          xAxis: [{
-            labels: {
-              enabled: false, // overwrites showLabels prop when condition is met
-            },
-          }],
-        },
-      }],
-    },
-  }
+  // maps patternClass.name to custom color keys defined in tailwind.config.js
+  //  tailwind doesn't support templated class names, hence we need to use this lookup
+  const backgroundColorClasses: any = {
+    "Rent": "bg-rent",
+    "Transfer": "bg-transfer",
+    "Administration": "bg-administration",
+    "Eligibility": "bg-eligibility",
+    "Security of tenure": "bg-security",
+    "Develop": "bg-develop",
+    "Stewardship": "bg-stewardship",
+    "Use": "bg-use",
+  };
+
+  // added to match figma marker designs, but colored text doesn't look great? 
+  const textColorClasses: any = {
+    "Rent": "text-rent",
+    "Transfer": "text-transfer",
+    "Administration": "text-administration",
+    "Eligibility": "text-eligibility",
+    "Security of tenure": "text-security",
+    "Develop": "text-develop",
+    "Stewardship": "text-stewardship",
+    "Use": "text-gray-400",
+  };
```

Comment:
We're going to need this elsewhere also - can it be in a style utils file or similar?

Reply:
Yep for sure, will make this change now before merging :+1:

Reply:
So I attempted to import these consts from `'lib/utils.ts`, but that actually raised the same build-time error as I was getting when attempting to template them - fiddled with for awhile, but ultimately leaving in this file for now!

Annoyingly I suspect we may just have to re-define these in each file where we want to use them.